functions/Restore-DbaBackup.ps1

```diff
@@ -0,0 +1,215 @@
+function Restore-DbaBackup
```

Comment:
@Stuart-Moore  yeah, at the time i thought there were some of overlapping ones. Please feel free to include those in this PR -- no file limit since they are all yours. My apologies.